local-tests/test_catch_up.py

```diff
@@ -33,45 +36,45 @@
 
 chain.set_flags_validator('validator')
 
-print('Starting the chain')
+printt('Starting the chain')
 chain.start('aleph')
 
-print('Waiting 90s')
-sleep(90)
-
-finalized_before_kill_per_node = check_finalized(chain)
-print('Killing one validator and one nonvalidator')
+printt('Waiting for finalization')
+chain.wait_for_finalization(0)
+printt('Waiting for authorities')
+chain.wait_for_authorities()
 
-chain[3].stop()
-chain[4].stop()
+printt('Killing one validator and one nonvalidator')
```

Comment:
how does this statement work with `nonvalidators=all_accounts[4:]` ?

Reply:
I don't understand :/ Nodes 0,1,2,3 are validators, while 4 and 5 are non-validators.

Reply:
I was always bad at indices and digits in general :disappointed: